code/game/objects/items/oddities.dm

```diff
@@ -599,7 +599,7 @@
 	)
 	price_tag = 1997
 
-	spawn_tags = SPAWN_TAG_ODDITY
+	spawn_blacklisted = TRUE
```

Comment:
You dfo

Reply:
You dfo

Reply:
spawners work weirdly. i think it might be able to spawn if it has spawn tags, the onestar toolmods work that way as they have a spawn tag but are blacklisted but still spawn. ill try testing it with the tag back to see if your right

Reply:
If it still spawns with the tag on just comment it out to avoid confusion if we (ever) add it back.

Reply:
there, i commented the tag out

Reply:
Did you test it or just decided to comment it out?

Reply:
> Did you test it or just decided to comment it out?

commented it out as for some godforsaken reason my byond takes half an hour to connect or launch something

Reply:
the version with the uncommented thing is still compiling and connecting

Reply:
well it just finished compiling and connecting and i think your right. it got no results